doc/about/01_history.rst

```diff
@@ -0,0 +1,37 @@
+History
+=======
+
+The Development of Pastas started in the spring of 2016 at the TU Delft (The Netherlands) and Artesia (The
+Netherlands). Researchers at the TU Delft required a flexible framework that could support future research on time
+series analysis of groundwater data. Consulting company `Artesia <https://www.artesia-water.nl>`_ on the other hand,
+needed a tool that allowed them to perform more complex analyses in a scripted environment. As such, Pastas was
+developed under a strong partnership between academia and private industry, a collaboration that continues until this
+day.
+
+.. figure:: ./../_static/history_initial.jpg
+    :figwidth: 600px
+
+    Drawing the initial design of Pastas on a whiteboard (5th of April, 2016)
+
+Python was chosen as the programming language for Pastas, being a flexible and open-source language that allows for
+quick prototyping. Moreover, many if not most hydrologists learn Python during their education, making the software
+available to many. From the start, all code was made completely open-source under MIT license, embracing open science
+and FAIR data practices.
+
+International use of Pastas and research grew substantially after the publication of the Pastas article in the
+international journal Groundwater :cite:p:`collenteur_pastas_2019`. In addition, the University of Graz in Austria
+and now the Swiss research institute Eawag employed one of the maintainers, further pushing the international use and
+development of Pastas. Since its inception, Pastas has been applied in over a dozen countries worldwide. Supporting
+international research continues to be an important goal of Pastas. For a list of peer-reviewed publications using
+Pastas please see the :doc:`Publications page <publications>`.
+
+Pastas 1.0
+----------
+
+In february 2023, Pastas version 1.0 was released, the celebration of 7 years of research and development of Pastas.
+Although Pastas was already operational for a couple of years, the application programming interface (API) was still
+changing sporadically and older (unused) model code was mounting. This made maintenance harder and harder. In Pastas
```

Comment:
harder and harder -> increasingly difficult?

Reply:
harder better faster stronger?

Reply:
see [4d26710](https://github.com/pastas/pastas/pull/502/commits/4d2671080dfc6660cf34aeb29c457440080e5530)